package.json

```diff
@@ -48,10 +48,13 @@
     "pretest": "yarn lint",
     "test": "node ./test/runTest.js",
     "test:update": "SNAPSHOT_UPDATE=true node ./test/runTest.js",
-    "dev": "NODE_ENV=dev node ./scripts/build.js",
-    "watch": "NODE_ENV=dev,watch node ./scripts/build.js",
-    "build": "NODE_ENV=prod node ./scripts/build.js",
-    "package": "yarn build && vsce package --yarn",
+    "dev:watch": "yarn dev --watch --sourcemap",
+    "build:dev": "yarn build:extension && yarn build:worker",
+    "build": "yarn build:extension --minify && yarn build:worker --minify",
```

Comment:
Should we enable `--sourcemap` for better debugging experience? @idahogurl

Reply:
@JounQin Depends if the sourcemap will be used when an error is thrown. I think we are fine without it.

Reply:
Sourcemap will be used when error thrown, it was a bit painful to debug what caused `import()` not working. If there are sourcemaps, when people report issues, we'll find the root cause much easier.

Reply:
@JounQin Let's add sourcemaps then.

Reply:
@idahogurl Do you want a new PR or you'll just add that?

Reply:
@JounQin I'll add it.